lib/src/test/kotlin/network/provider/ProviderTest.kt

```diff
@@ -54,94 +54,40 @@ class ProviderTest {
         assumeTrue(NetworkConfig.isTestEnabled(requiresGas = false))
 
         val transactionHash = when (network) {
-            Network.GOERLI_INTEGRATION -> Felt.fromHex("0x26396c032286bcefb54616581eea5c7e373f0a21c322c44912cfa0944a52926")
+            Network.GOERLI_INTEGRATION -> Felt.fromHex("0x34223514e92989608e3b36f2a2a53011fa0699a275d7936a18921a11963c792")
             Network.GOERLI_TESTNET -> Felt.fromHex("0x72776cb6462e7e1268bd93dee8ad2df5ee0abed955e3010182161bdb0daea62")
-            else -> throw NotImplementedError("Sepolia networks are not yet supported")
+            Network.SEPOLIA_INTEGRATION -> Felt.fromHex("0x3d381262ca26570083eab24e431b72e69ce5e33b423c034aef466613ee20511")
+            Network.SEPOLIA_TESTNET -> Felt.fromHex("0x03f6e39d90aa084585ac59fa4c19429499951eb7a279427bbe0a7718c8d9072f")
         }
         val transactionHash2 = when (network) {
             Network.GOERLI_INTEGRATION -> Felt.fromHex("0x5e2e61a59e3f254f2c65109344be985dff979abd01b9c15b659a95f466689bf")
             Network.GOERLI_TESTNET -> Felt.fromHex("0x6bf08a6547a8be3cd3d718a068c2c0e9d3820252935f766c1ba6dd46f62e05")
-            else -> throw NotImplementedError("Sepolia networks are not yet supported")
+            Network.SEPOLIA_INTEGRATION -> Felt.fromHex("0xe691cd69a7dc7efb815f6029db26cf5d7a14f2b5e893e2de4a281c82b5a0d")
+            Network.SEPOLIA_TESTNET -> Felt.fromHex("0x7246ea24cefc572096264f1f243cd9b32837c17324ac23076070f29f50b6393")
         }
         val transactionStatus = provider.getTransactionStatus(transactionHash).send()
         // TODO: Re-enable this assertion for integration once transaction appear as accepted on L1 again
         if (network != Network.GOERLI_INTEGRATION) {
-            assertEquals(TransactionFinalityStatus.ACCEPTED_ON_L1, transactionStatus.finalityStatus)
+            assertEquals(TransactionStatus.ACCEPTED_ON_L1, transactionStatus.finalityStatus)
         }
         assertNotNull(transactionStatus.executionStatus)
         assertEquals(TransactionExecutionStatus.SUCCEEDED, transactionStatus.executionStatus)
 
         val transactionStatus2 = provider.getTransactionStatus(transactionHash2).send()
-        // TODO: Re-enable this assertion for integration once transaction appear as accepted on L1 again
-        if (network != Network.GOERLI_INTEGRATION) {
-            assertEquals(TransactionFinalityStatus.ACCEPTED_ON_L1, transactionStatus2.finalityStatus)
-        }
+
         assertNotNull(transactionStatus2.executionStatus)
         assertEquals(TransactionExecutionStatus.REVERTED, transactionStatus2.executionStatus)
     }
 
-    @Disabled
-    @Test
-    fun `estimate message fee`() {
-        assumeTrue(NetworkConfig.isTestEnabled(requiresGas = false))
```

Comment:
I decided there's no good reason to spend time updating this test, so I removed it altogether.
Network tests are meant for basic standard flow operations and new features. This test was created when there was no devnet with new rpc version, now this functionality is tested fine on devnet. If there are changes to it, it would be easier to write a new test case from scratch than redo this one.